### PR TITLE
Change Package Manager 

### PR DESCRIPTION
**Pull request by Mr-MRF-Dev**
closed: #30

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -1,27 +1,108 @@
...
-    // Best practices
-    "strict": true,
-    "skipLibCheck": true,
-    "noFallthroughCasesInSwitch": true,
-
-    // Some stricter flags (disabled by default)
-    "noUnusedLocals": false,
-    "noUnusedParameters": false,
-    "noPropertyAccessFromIndexSignature": false
+    /* Visit https://aka.ms/tsconfig to read more about this file */
```
Comment (mkermani144) [inline tsconfig.json:3]: Do not leave helper comments generated by a tool in production code. All of the configs and their functionality is available in Typescript official docs.  

In addition, this default `tsconfig` is not a good one for our project. Please add a new issue for updating it.

**Review by mkermani144 — CHANGES_REQUESTED** [review]

File: tsconfig.json (hunk 1 above)
Comment (Mr-MRF-Dev) [reply tsconfig.json]: done 🍀

File: config.ts [hunk 2]
```diff
@@ -1,5 +1,8 @@
 // import env file with 'dotenv'
-require('dotenv').config({ path: '.env' });
+// require('dotenv').config({ path: '.env' });
```
Comment (mkermani144) [inline config.ts:2]: Don't leave commented out code in production.

File: config.ts [hunk 3]
```diff
@@ -1,5 +1,8 @@
...
+
+import dotenv from 'dotenv';
+dotenv.config({ path: '.env' });
```
Comment (mkermani144) [inline config.ts:5]: We should at least support `.env` and `.env.local`.

File: tsconfig.json [hunk 4]
```diff
@@ -1,108 +1,17 @@
...
-    // "noPropertyAccessFromIndexSignature": true,       /* Enforces using indexed accessors for keys declared using an indexed type. */
-    // "allowUnusedLabels": true,                        /* Disable error reporting for unused labels. */
-    // "allowUnreachableCode": true,                     /* Disable error reporting for unreachable code. */
-
-    /* Completeness */
-    // "skipDefaultLibCheck": true,                      /* Skip type checking .d.ts files that are included with TypeScript. */
-    "skipLibCheck": true /* Skip type checking all .d.ts files. */
-  }
+    "module": "ESNext",
+    "target": "es6",
```
Comment (mkermani144) [inline tsconfig.json:4]: Let's use `esnext` for this one, too. We want to use the latest features of Javascript in the generated code.

File: tsconfig.json [hunk 5]
```diff
@@ -1,108 +1,17 @@
...
+    "esModuleInterop": true,
+    "skipLibCheck": true,
+    "forceConsistentCasingInFileNames": true,
+    "strict": true,
+    "moduleResolution": "node",
```
Comment (mkermani144) [inline tsconfig.json:9]: Use `node16` (or `nodenext`). There are some additional features that we may need.  
You may also use `bundler`, but I'm not sure it's a good fit for our project, at least for now.

**Review by mkermani144 — CHANGES_REQUESTED** [review]

**Review by mkermani144 — APPROVED** [review]

**Review by mkermani144 — CHANGES_REQUESTED** [review]
Please resolve Git conflicts.

**Review by mkermani144 — APPROVED** [review]

**Pull request merged**